sqlglot/dialects/tsql.py

```diff
@@ -200,6 +200,17 @@ def inner_func(args: t.List) -> E:
     return inner_func
 
 
+def setitem_sql(self, expression: exp.SetItem) -> str:
```

Comment:
just put this in the generator

Reply:
It's a little crowded but got it down to a one liner